tests/integration/customer/test_utils.py

```diff
@@ -13,3 +13,7 @@ def test_dispatcher_password_reset_url(self):
         user = User.objects.create_user('testuser', 'text@example.com', 'dummypassword')
         url = get_password_reset_url(user)
         self.assertTrue(url.startswith('/password-reset/confirm/'))
+
+    def test_normalise_email(self):
+        self.assertEqual(normalise_email(None), '')
```

Comment:
Why should we accept a `None` value here? I don't think it is unreasonable for the function to expect a string as opposed to any other type.